### ISSUE TITLE
Changes in ESPN coding.

### ISSUE THREAD

**Issue by Bernard-h** [jclarke0000/MMM-MyScoreboard#50]
Ive seen the recent posts in regards to some leagues not working (AAA baseball, English premier league). I cannot add the new european football cup: UEFA CONFERENCE LEAGUE. So I contacted ESPN and asked if they would add it to their website. And.... they did... so I tried the new league code 

  {
        league: "UEFA_EUROPA_CONF",
	teams: ["AJAX", "PSV", "FEYE", "VIT", "UTR", "ALK"],
	label: "CONF"   
}, 

and it doesnt work. I checked the league code on the espn.com website and its correct. So apparently something in the coding must be added to pull the data. Has anybody out here any idea on how to add that to the coding of the module? 

Perhaps JCLARKE000? 

Anyway, thanks for the help.

**Comment by Bernard-h** [comment]
Its pretty easy actually.... edit 2 files inside the module folder and insert your league information. I added the europa conference league and only noticed 1 thing. The teams codes are different for the conference league which is weird... but it works if you tweak it. 

this is the magicmirror config:

{
        league: "UEFA_EUROPA_CONF",
	teams: ["AJAX", "PSV", "FEY", "VIT", "UTR", "AZ"],
	label: "COL"        
      },
	{
        league: "UEFA_CHAMPIONS",
	teams: ["AJAX", "PSV", "FEY", "VIT", "UTR", "ALK"],
	label: "CL"        
      },

NOTICE THAT the same teams have different teamcodes.

The two files you edit are: 
MMM-Myscoreboard.js in the root directory ... add you league there and use the format of all the dozens of other leagues as example. In my case I added: 
 "UEFA_EUROPA_CONF": {provider: "ESPN", logoFormat: "url", homeTeamFirst: true},

2nd open the file ESPN.js and add your league there. I added:
"UEFA_EUROPA_CONF": "soccer/uefa.europa.conf",

When I first ran with these additions, only 1 team showed in stead of 3... so I kinda knew right away the problem was teamcodes. So I fiddeled around logically and found my teams very quickly.
